kafka-rest/src/main/java/io/confluent/kafkarest/tools/ProducerPerformance.java

```diff
@@ -104,7 +104,7 @@ protected void doIteration(PerformanceStats.Callback cb) {
       URL url = new URL(targetUrl);
       connection = (HttpURLConnection) url.openConnection();
       connection.setRequestMethod("POST");
-      connection.setRequestProperty("Content-Type", Versions.KAFKA_MOST_SPECIFIC_DEFAULT);
```

Comment:
@stanislavkozlovski , @ewencp : do you know why KAFKA_MOST_SPECIFIC_DEFAULT is still KAFKA_V1_JSON?

Reply:
I do not know why that is. I see that it is used extensively throughout our tests so changing it might not be trivial and require us to update a lot of tests (while also re-introduce ones for V1)

Reply:
it's just because that's all that existed when this was written and I don't think updating this got scoped into the v2 effort